using-timescaledb/continuous-aggregates.md

```diff
@@ -18,118 +18,162 @@ re-calculate all of the data on every refresh. Only new and/or
-hypertable. For example, you can create a continuous aggregate
-`condition_summary_daily` over the data with daily buckets:
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary_daily',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '1 h');
+```
+
+In this case, the continuous aggregate will be refreshed every hour
+and refresh the last month's data.
+
+You can now run a normal `SELECT` on the continuous aggregate and it
+will give you the aggregated data, for example, to select the averages
+for device 1 during the first three months:
 
 ```sql
-CREATE VIEW conditions_summary_daily
+SELECT bucket, avg FROM conditions_summary_daily
+WHERE device = 1 AND bucket BETWEEN '2020-01-01' AND '2020-03-31'
+ORDER BY bucket;
+```
+
+### A detailed look at continuous aggregate views [](detailed-look)
+
+As shown above, creating a refreshing [continuous
+aggregate][api-continuous-aggs] is a two-step process. First, one
+needs to create a continuous aggregate view of the data using [`CREATE
+MATERIALIZED VIEW`][postgres-createview] with the
+`timescaledb.continuous` option. Second.  a continuous aggregate
+policy needs to be created to keep it refreshed.
+
+You can create several continuous aggregates for the same
+hypertable. For example, we could create another continuous aggregate
+view that summarizes the hourly data.  updated hourly, you would
+write:
+
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_hourly
 WITH (timescaledb.continuous) AS
 SELECT device,
-       time_bucket(INTERVAL '1 day', time) AS bucket,
+       time_bucket(INTERVAL '1 hour', time) AS bucket,
        AVG(temperature),
        MAX(temperature),
        MIN(temperature)
 FROM conditions
 GROUP BY device, bucket;
+
+-- Create the policy
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => INTERVAL '1 month',
+	end_interval => NULL,
+	schedule_interval => INTERVAL '5 min');
 ```
 
+A `time_bucket` on the time partitioning column of the hypertable is
+required in all continuous aggregate views. If you do not provide one,
+you will get an error.
+
+When the view is created, it will (by default) be populated with data
+so that it contain an aggregate of the current data in
+`conditions`. If you make changes to the hypertable and want to
+refresh an interval of the continuous aggregate, you can explicity
+refresh the continuous aggregate by calling
+[`refresh_continuous_aggregate`][refresh_continuous_aggregate] and
+give the range you want to refresh:
+
+```sql
+CALL refresh_continuous_aggregate('conditions_summary_daily, '2020-05-01', '2020-06-01');
+```
+
+This will update the buckets in the range that were invalidated by
+changes in the underlying hypertable, but not touch other buckets of
+the continuous aggregate, so it does not necessarily mean that all
+buckets in the range are touched. If there are just a few
+invalidations in the range, the refresh will be quick.
```

Comment:
I would not start talking about invalidations here as it doesn't seem necessary to convey the main message (i.e., that refreshes are efficient). Instead, I suggest talking about how things work in the abstract.

```suggestion
Unlike a regular materialized view, the refresh will only materialize the data within the window that has changed in the underlying hypertable since the last refresh. Therefore, if only a few buckets need updating, then the refresh is quick.
```